api-tests/features/inherited-identity/extended-scenarios.feature

```diff
@@ -0,0 +1,290 @@
+    Then I get a '<expected-identity>' identity
+
+    Examples:
+    | inherited-identity | identity-details             | expected-identity |
+    | PCL200             | alice-vot-pcl200-no-evidence | PCL200            |
+    | PCL250             | kenneth-vot-pcl250-passport  | PCL250            |
+
+  Scenario Outline: Successful enhanced verification mitigation with a <inherited-identity> HMRC profile and receives a P2
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I get an 'access_denied' OAuth error from the CRI stub
+    Then I get a 'page-multiple-doc-check' page response
+    When I submit a 'ukPassport' event
+    Then I get a 'ukPassport' CRI response
+    When I submit 'kenneth-passport-valid' details to the CRI stub
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'page-pre-experian-kbv-transition' page response
+    When I submit a 'next' event
+    Then I get a 'kbv' CRI response
+    When I submit 'kenneth-needs-enhanced-verification' details to the CRI stub
+    Then I get a 'pyi-suggest-other-options' page response
+
+    # New journey with inherited identity - user still needs to mitigate the CI
+    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity '<identity-details>'
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I get an 'access_denied' OAuth error from the CRI stub
+    Then I get a 'pyi-post-office' page response
+    When I submit a 'next' event
+    Then I get a 'claimedIdentity' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'f2f' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the async CRI stub that mitigate the 'NEEDS-ENHANCED-VERIFICATION' CI
+    Then I get a 'page-face-to-face-handoff' page response
+
+    # Return journey
+    When I start a new 'medium-confidence' journey and return to a 'page-ipv-reuse' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P2' identity
+
+    Examples:
+    | inherited-identity | identity-details             |
+    | PCL200             | alice-vot-pcl200-no-evidence |
+    | PCL250             | kenneth-vot-pcl250-passport  |
+
+  Scenario Outline: Fails to migrate <inherited-identity> HMRC profile when user has failed F2F with CI
```

Comment:
Do you know why this is specifically for F2F? Feels like it would be a quicker/easier test to use DCMAW

I think the important part of these tests is checking that you get appropriate behaviour when migrating an inherited ID with a CI, i.e.
- BREACHING -> no match
- ENHANCED_VERIFICATION_REQUIRED -> enhanced verification mitigation journey
- ALTERNATE_DOC_REQUIRED -> alternate doc mitigation journey

Reply:
When You inherit Are you not checking which route it failed if not then Its ok, We have these cases in selenium tests as we seen failures in different routes at times so covering all the cases

Reply:
Hmm, I suppose there are a few different things that could conflict with each other here. Of course it _shouldn't_ matter which journey if you've got a CI.

Maybe let's revisit when we pick up inherited identities again?